app/views/users/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>All Users</title>
+</head>
+<body>
+  <body>
+  <h1><%= @specific_user.name %></h1>
+  <p>Bio: <%= @specific_user.bio %></p>
+  <h2>Posts by <%= @specific_user.name %></h2>
+  <ul>
+    <% @user_post.each do |post| %>
+      <li><%= post.title %></li>
+      <p>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %></p>
+    <% end %>
+  </ul>
+  
+</body>
```

Comment:
- [ ] Per the requirement we should only have placeholder texts. Kindly add only placeholder text as required.